samples/compose/src/main/java/com/squareup/sample/compose/hellocomposebinding/HelloBinding.kt

```diff
@@ -0,0 +1,31 @@
+package com.squareup.sample.compose.hellocomposebinding
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.squareup.sample.compose.hellocomposebinding.HelloWorkflow.Rendering
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.compose.composeViewFactory
+
+@OptIn(WorkflowUiExperimentalApi::class)
+val HelloBinding = composeViewFactory<Rendering> { rendering, _ ->
+  Text(
+    rendering.message,
+    modifier = Modifier
+      .fillMaxSize()
+      .clickable(onClick = rendering.onClick)
+      .wrapContentSize(Alignment.Center)
```

Comment:
For my own education, what's the rationale behind calling `fillMaxSize()` and `wrapContentSize()`. I would've thought they'd be mutually exclusive.

(super nit if you're okay relying on the default: `Alignment.Center` is already the default arg for `wrapContentSize()`.)

Reply:
`wrapContentSize` centers the text inside the full-screen clickable.